staging/src/k8s.io/apiextensions-apiserver/test/integration/basic_test.go

```diff
@@ -80,6 +81,246 @@ func TestClusterScopedCRUD(t *testing.T) {
 	testFieldSelector(t, ns, noxuDefinition, dynamicClient)
 }
 
+func TestVersionedNamspacedScopedCRD(t *testing.T) {
```

Comment:
please move all versioning tests into versioning_test.go. This file is getting too big.